frontend/supabase/migrations/20250623182937_remote_schema.sql

```diff
@@ -0,0 +1,6765 @@
+
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+
+CREATE SCHEMA IF NOT EXISTS "gis";
+
+
+ALTER SCHEMA "gis" OWNER TO "postgres";
+
+
+CREATE EXTENSION IF NOT EXISTS "pgsodium";
+
+
+
+
+
+
```

Comment:
weird there's so much whitespace!

Reply:
Now it's only two newlines when I pulled again using a different flag